Fastpack/Fastpack.ml

```diff
@@ -226,19 +227,34 @@ let prepare_and_pack options start_time =
   Lwt.finalize
     (fun () ->
        let%lwt {Reporter. graph; _} = init_run () in
-       match options.watch with
-       | false ->
+       match (options.watch, options.serve) with
```

Comment:
This solution is kind of ugly, I think we should assume that if `serve` is set then the user is fine with `watch` and `development` being set as well.